packages/router/src/grpc/view-protocol-server/index-by-address.ts

```diff
@@ -17,11 +17,7 @@ export const indexByAddress: Impl['indexByAddress'] = async (req, ctx) => {
 
   const addressIndex = isControlledAddress(fullViewingKey, address);
 
-  if (!addressIndex)
-    throw new ConnectError(
-      'Address is not controlled by view service full viewing key',
-      Code.Unauthenticated,
-    );
+  if (!addressIndex) return {};
```

Comment:
We should not return an error if the address is not associated with our FVK, but instead return an empty response like rust does.
My guideline is that the `extension view service` and the `rust viewer service` should have the same behavior and be interchangeable.

Reply:
I don't know if I'm the best person to make the call on whether this reasoning is OK, but it sounds fine to me :D

Reply:
Normally, I'd disagree, but it appears the proto definitions have this comment:

```
  /**
   * Will be absent if given an address not viewable by this viewing service
   *
   * @generated from field: penumbra.core.keys.v1alpha1.AddressIndex address_index = 1;
   */
  addressIndex?: AddressIndex;
```
So I suppose this is the expected behavior